source/server-selection/server-selection.rst

```diff
@@ -125,6 +125,10 @@ Terms
     Describes candidate servers that also meet the criteria specified by the
     ``tag_sets`` and ``maxStalenessSeconds`` read preference parameters.
 
+**Hedged Read**
+    A server mode in which the same query is dispatched in parallel to multiple
+    replica set members.
```

Comment:
I'll note that drivers will rely on a 4.4 mongos to perform dispatching, so this presently only applies to sharded clusters where shards are replica sets. I expect that will change down the line if drivers ultimately need to handle concurrent dispatch when connected to a replica set.

I don't think this text needs to be changed, but I wanted to point it out for the benefit of other reviewers.